sdk/nodejs/src/models/populatedUserHelpers.ts

```diff
@@ -0,0 +1,99 @@
+import { ProtobufTypes } from '@devcycle/bucketing-assembly-script'
+import {
+    DVCPopulatedUser,
+    DevCycleUser,
+    DVCCustomDataJSON,
+} from '@devcycle/js-cloud-server-sdk'
+import { getNodeJSPlatformDetails } from '../utils/platformDetails'
+
+export function DVCPopulatedUserToPBUser(
```

Comment:
These were the Protobuf instance methods extracted from the previous `DVCPopulatedUser.toPBUser()` methods.